rts/Menu/LuaMenuController.cpp

```diff
@@ -80,6 +79,9 @@ bool CLuaMenuController::Draw()
 	// calls IsAbove
 	mouse->GetCurrentTooltip();
 
+	
+	spring_msecs(10).sleep(true);
```

Comment:
why did you move the location of the sleep?
(I'm quite certain it should be where it was)